CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ a release.
 
 ## [Unreleased]
 
+### Fixed
+- Sortable: Fix return value check of Comparable interface (#2541)
+
+### BC break
```

Comment:
As we are not removing the support for other types, there is no BC break here.

Reply:
@phansys strictly speaking if someone would return `1` and expect it to be evaluated as `truthy` it would fail now as it would result in `false` which strictly speaking is a change from before. 

However, it is an edge case as someone used the correct return type but invalid behavior which makes no sense.